services/order.py

```diff
@@ -0,0 +1,30 @@
+from django.db import transaction
+from datetime import datetime
+from db.models import Order, Ticket, User
+from db.models import MovieSession
+
+
+def create_order(tickets: list, username: str, date: str = None) -> User:

```

Comment:
The return type annotation should be `Order` instead of `User` since the function returns an `Order` object.